nautobot_ssot/models.py

```diff
@@ -47,13 +47,18 @@ class Sync(BaseModel):
 
     start_time = models.DateTimeField(blank=True, null=True)
     # end_time is represented by the job_result.completed field
-    load_time_source = models.DurationField(blank=True, null=True)
-    load_time_target = models.DurationField(blank=True, null=True)
+    source_load_time = models.DurationField(blank=True, null=True)
+    target_load_time = models.DurationField(blank=True, null=True)
     diff_time = models.DurationField(blank=True, null=True)
     sync_time = models.DurationField(blank=True, null=True)
-    memory_usage = models.IntegerField(blank=True, null=True)
-    memory_size = models.IntegerField(blank=True, null=True)
-    memory_peak = models.IntegerField(blank=True, null=True)
+    source_load_memory_final = models.PositiveBigIntegerField(blank=True, null=True)
+    source_load_memory_peak = models.PositiveBigIntegerField(blank=True, null=True)
+    target_load_memory_final = models.PositiveBigIntegerField(blank=True, null=True)
+    target_load_memory_peak = models.PositiveBigIntegerField(blank=True, null=True)
+    diff_memory_final = models.PositiveBigIntegerField(blank=True, null=True)
+    diff_memory_peak = models.PositiveBigIntegerField(blank=True, null=True)
+    sync_memory_final = models.PositiveBigIntegerField(blank=True, null=True)
+    sync_memory_peak = models.PositiveBigIntegerField(blank=True, null=True)
```

Comment:
Would it be better to default these to 0 instead of allowing them to be blank/null? Not sure.

Reply:
I thought about defaulting to `null` to be more explicit about data that has not been gathered at all and avoid confusing with a execution that is not actually consuming any memory (i.e. a simple `pass`) so we can easily understand if the data was actually the "default" vs something actually gathered.
But maybe you have some other point in mind that I am not considering?

Reply:
No, that's a fair point. Thanks.